### DISCUSSION TITLE
Provide an obfuscated metadata file that includes the "time" field

### DISCUSSION THREAD

**Discussion by zkochan** [verdaccio/verdaccio#3315] [Ideas]
I was working on [a new resolution mode](https://github.com/pnpm/pnpm/pull/5238) for pnpm. You can read about this idea in the [this RFC](https://github.com/pnpm/rfcs/pull/2). The new resolution mode works well but it is slower because it needs to fetch the full metadata. It would be great to experiment with Verdaccio by adding support to a new metadata format to Verdaccio. Currently the public npm registry can return full metadata and compact metadata (related issue about it in verdaccio: https://github.com/verdaccio/verdaccio/issues/2792). I would like Verdaccio to support a 3rd metadata format, it would be the same as the compact one but would also include the "time" field from the full metadata.

If you could support it, we would be able to validate our new resolution mode in pnpm. It could be also good PR for Verdaccio as it would support a new feature sooner than the public registry.

**Comment by juanpicado** [comment]
I think is doable, I was about to start on https://github.com/verdaccio/verdaccio/issues/2792 for the v6, not sure about touch v5 (I'm trying not to) ,  does a v6 alpha implementation would help you keep experimenting with the new resolution mode?

v6 is already available via npmjs https://github.com/verdaccio/verdaccio/discussions/2970 

```
npm install --global verdaccio@6-next
```

**Reply by zkochan** [reply]
Even just a branch with a new endpoint would be enough for me.

**Reply by juanpicado** [reply]
follow up here https://github.com/verdaccio/verdaccio/pull/3322
